Vizzuality/half-earth-v3
# Feature/aoi wdpas

## Add insect richness layers. Specific regions and new protected areas layers
### Description
- Add insect richness layers to Data globe (Missing metadata)
- Add Specific regions (under REACT_APP_FEATURE_SPECIFIC_REGIONS_AOI tag) Mississippi region is freezing
- Add new Protected areas layers for it to load faster
### Testing instructions

- Go to Biodiversity - Richness - Terrestrial species dropdown -> Country resolution. You will find the new layers there
- Go to Analyse areas - Select specific regions - click on one of them (Mississippi will freeze)
- Go to Analyse areas - Select Protected areas - You should see different protected areas resolutions depending on the zoom level
 
### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-384?atlOrigin=eyJpIjoiM2I5NWVkMzAxN2IxNGFkMWIyN2I0MWNkZWQxMDIxNjIiLCJwIjoiaiJ9
https://vizzuality.atlassian.net/browse/HE-384?atlOrigin=eyJpIjoiZTk3ZWM1ODliN2MxNDI2ZDk3MDY1YzhmYThkYmRiNWMiLCJwIjoiaiJ9
https://vizzuality.atlassian.net/browse/HE-387?atlOrigin=eyJpIjoiYmUzOWIzNjk5OWIyNDQ0Y2I4YjQ0ZGFlZjA5NjRkYTEiLCJwIjoiaiJ9
